### DISCUSSION TITLE
Illegal Characters In Attributes

### DISCUSSION THREAD

**Discussion by fweth** [alpinejs/alpine#3128] [4. General]
Hi, I just used the `<` character in some JS code for Alpine in a HTML attribute, and VScode marked it red, but still seemed to work and the browser didn't complain.  I looked up the [spec](https://html.spec.whatwg.org/multipage/syntax.html#attributes-2) and interestingly, they claim that just the other way around, `>` is not valid in HTML attributs.  So anyway, I just wondered, could this be a problem for some browsers?  They also claim `=` is not valid, which would make writing JS for Alpine really hard.

**Comment by SimoTod** [comment]
They are talking about the name of the attribute. `<tag name="value">`
The `=` is used by the browser to separate the name and the value.
What is your use case for having an `=` in an attribute name?

**Reply by fweth** [reply]
Hey, thanks for the answer, maybe I didn't understand the spec correctly.  I just had something like `x-show="a < b"` and VScode marked the `<` in red, so I wondered, if there are any characters which I chouldn't use inside Alpine attributes.

**Reply by SimoTod** [reply]
except for the double quotes symbol which conflicts with the value terminator, I'm not aware of any other restrictions.

**Reply by fweth** [reply]
OK, thanks a lot!
